src/App.tsx

```diff
@@ -2,11 +2,13 @@ import { Route, Routes } from "react-router-dom";
 import "./App.css";
 import Login from "./Pages/Login";
 import Register from "./Pages/Register";
+import Home from "./Pages/Home";
 
 function App() {
   return (
     <div className="App">
       <Routes>
+        <Route path="/" element={<Home />} />
         <Route path="/login" element={<Login />} />
         <Route path="/register" element={<Register />} />
       </Routes>
```

Comment:
Suggestion: can create an AppRoutes.js file and import from there as in the future no. of routes will increase so will help in making code more readable here.